src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -82,7 +82,7 @@ public static List<PartyDetails> buildHearingPartiesDetails(SscsCaseData caseDat
             for (CcdValue<OtherParty> ccdOtherParty : otherParties) {
                 OtherParty otherParty = ccdOtherParty.getValue();
                 partiesDetails.addAll(buildHearingPartiesPartyDetails(
-                        otherParty, otherParty.getRep(), otherParty.getHearingOptions(), null, otherParty.getHearingSubtype(), appellant.getId()));
+                        otherParty, otherParty.getRep(), otherParty.getHearingOptions(), appeal.getHearingType(), otherParty.getHearingSubtype(), appellant.getId()));
```

Comment:
otherParty shouldn't be using appeal.getHearingType(), that is for the appellant.

Reply:
Being addressed here: https://github.com/hmcts/sscs-hearings-api/pull/185